roles/pulibrary.nginxplus/ADDHOSTS.md

```diff
@@ -0,0 +1,39 @@
+Adding new hosts to Nginx Plus
+==============================
+
+* You will need to register the new Virtual Server at
+  [https://networkregistration.princeton.edu](https://networkregistration.princeton.edu) - for now this is a bug because permission/authority is required from OIT, so
+ping the #Operations Slack channel. The turn around is instant + time for DNS to
+propagate worldwide.
+
+* Generate SSL keys for the Virtual Server above. This is also another bug that
+  will be fixed with a ping to the #Operations Slack channel. The turn around
+  here averages about 12 hours.
+
+* You will need at least one upstream server (for staging) and we encourage at
```

Comment:
This really may be my lack of understanding of the process, but what is the step you want done here?  Could all the operations pings on the slack channel be done in one step?  An example of the information included int eh slack ping here might also help.

Reply:
yeah, this makes sense. It should be one step.